### ISSUE TITLE
IDISP009 false positive for IHttpModule

### ISSUE THREAD

**Issue by kuzyaka** [DotNetAnalyzers/IDisposableAnalyzers#80]
My class implemented System.Web.IHttpModule with the signature
```cs
namespace System.Web
{
  public interface IHttpModule
  {
    void Init(HttpApplication context);
    void Dispose();
  }
}
```

I guess IDisposableAnalyzers should not warn in this case.

**Comment by JohanLarsson** [comment]
I dumbed this one, fixing, thanks for reporting!
